src/test/java/com/puppycrawl/tools/checkstyle/AbstractModuleTestSupport.java

```diff
@@ -218,6 +218,46 @@ protected final void verifyFilterWithInlineConfigParser(String filePath,
         verify(configWithFilters, filePath, expectedFiltered);
     }
 
+    /**
+     * Performs verification of two files with their given file paths using specified
+     * configuration of one file only. Also performs verification of the config with filters
+     * specified in the input file.
+     *
+     * @param filePath1 file path of first file to verify
+     * @param filePath2 file path of second file to verify
+     * @param expectedUnfiltered an array of expected unfiltered config.
+     * @param expectedFiltered an array of expected config with filters.
+     * @throws Exception if exception occurs during verification process.
+     */
+    protected final void verifyFilterWithInlineConfigParser(String filePath1,
```

Comment:
Trying to add this new helper method and use it but facing issues

```
Files with expected violations and actual violations differ.
value of      : keySet()
unexpected (1): /home/stoyan/Open Source/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterDefaultConfig2.java
---
expected      : [/home/stoyan/Open Source/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterDefaultConfig.java]
but was       : [/home/stoyan/Open Source/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterDefaultConfig2.java, /home/stoyan/Open Source/checkstyle/src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppresswithnearbytextfilter/InputSuppressWithNearbyTextFilterDefaultConfig.java]

	at com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport.verify(AbstractModuleTestSupport.java:408)
	at com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport.verify(AbstractModuleTestSupport.java:380)
	at com.puppycrawl.tools.checkstyle.AbstractModuleTestSupport.verifyFilterWithInlineConfigParser(AbstractModuleTestSupport.java:247)
	at com.puppycrawl.tools.checkstyle.filters.SuppressWithNearbyTextFilterTest.testTwoFilesOneAfterAnother(SuppressWithNearbyTextFilterTest.java:200)
	at [[Reflective call: 2 frames collapsed (https://goo.gl/aH3UyP)]].(:0)
	at [[Testing framework: 27 frames collapsed (https://goo.gl/aH3UyP)]].(:0)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at [[Testing framework: 9 frames collapsed (https://goo.gl/aH3UyP)]].(:0)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at [[Testing framework: 21 frames collapsed (https://goo.gl/aH3UyP)]].(:0)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)

```

Wrote it in the same way `verifyWithInlineConfigParser(file1, file2, expected)` is written. Strange how `JavadocPackageCheckTest:testWithMultipleFiles` works when using it but my method does not.